python/example_code/iam/create_role.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License. 
+# snippet-start:[iam.python.create_user.complete]

```

Comment:
This should say 
`#snippet-start:[iam.python.create_role.complete]`

Reply:
Hmm, copied it from existing IAM create_user example. :( Fixed in latest commit.